libraries/core/src/main/kotlin/com/bumble/appyx/core/node/ParentNodeView.kt

```diff
@@ -2,9 +2,17 @@ package com.bumble.appyx.core.node
 
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import com.bumble.appyx.core.plugin.NodeAware
 
-interface ParentNodeView<NavTarget : Any> : NodeView, NodeAware<ParentNode<NavTarget>> {
+interface ParentNodeView<NavTarget : Any> : NodeView {
+
     @Composable
     fun ParentNode<NavTarget>.NodeView(modifier: Modifier)
+
+    @Suppress("UNCHECKED_CAST")
+    @Composable
+    override fun View(modifier: Modifier) {
```

Comment:
is this okay if it gets overriden? or should it be final?

Reply:
Ideally it should be final, the problem is that `NodeVew` is an inteface

Reply:
Maybe add some JavaDoc specifying that overriding this function will cause unexpected behaviours?